v0/destinations/am/transform.js

```diff
@@ -84,15 +84,27 @@ const aliasEndpoint = destConfig => {
   return retVal;
 };
 
-function getSessionId(payload) {
-  const sessionId = payload.session_id;
-  if (sessionId) {
-    if (typeof sessionId === "string") {
-      return sessionId.substr(sessionId.lastIndexOf(":") + 1, sessionId.length);
-    }
-    return sessionId;
+function handleSessionIdUnderRoot(message) {
+  const sessionId = get(message, "session_id");
+  if (typeof sessionId === "string") {
+    return sessionId.substr(sessionId.lastIndexOf(":") + 1, sessionId.length);
```

Comment:
Can't we simply provide 0 apart from `sessionId.lastIndexOf(":") + 1`?

Reply:
So, the reason why we are starting from the `lastIndexOf(":)`, is because when the sessionId in the payload is in the format `"User:123456"`,  we want to consider only `123456`